app/javascript/packs/application.js

```diff
@@ -38,6 +38,10 @@ document.addEventListener('DOMContentLoaded', () => {
 
   $(document).on('click', 'body', logClick)
   $(document).on('change', 'select', logMenuChange)
+  $(document).on('copy', 'body', logCopy)
+  // contextmenu event is to handle when users use context menu "copy email address" instead of cmd+C copy event as
+  // this does not emit the copy event
+  $(document).on('contextmenu', '#zendesk-alert-email', logCopy)
```

Comment:
instead of hardcoding this to the id, I'd create a new functional css class like "log-on-context-copy" or somesuch.  That way we don't have to update this application.js file each time we want to do something like this

Reply:
Let's log all `contextmenu` events (on `body`, not any more particular selector) in a `logContextMenu` function like `logClick` or `logCopy`.

That would solve two things:
1. Opening the context menu does not necessarily entail selecting "Copy email address" (even on `mailto` links).
2. Additional, not-yet-known use cases for wanting to know when users open their context menu.

Regarding (1), users can open context menu then select "Copy", which emits a `copy` event, and the current instrumentation double counts copies in that.  Logging `contextmenu` separately would us have refined downstream handling for that in Mixpanel queries.

Reply:
> Regarding (1), users can open context menu then select "Copy", which emits a copy event, and the current instrumentation double counts copies in that case.

My testing has shown this does not in fact happen - selecting the "Copy" option from a context menu does not emit a copy event.  The JS copy event is only emitted by `cmd + C`.  I think it has something to do with the `contextmenu` event taking precedence during the right-click interaction.

Reply:
390a54e

Reply:
Hmm, I still see `copy` events logged to Mixpanel when selecting the "Copy" option from a context menu.  But my blocking concern is resolved, because I verified your `contextmenu` handler change addresses my main point!

(If you confirm that "Copy" via context menu _doesn't_ log, then that strikes me as a bug, but I can't reproduce it.)

Reply:
Ah - I see what is happening.  There is a distinction between selecting "Copy" and "Copy email address".  The former emits the `copy` event, and the latter does not.  I was always using "Copy email address".  I don't believe that there is a way for us to prevent this from happening as this is all native browser behavior.